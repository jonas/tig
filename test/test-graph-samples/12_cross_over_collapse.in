commit A B D E
    Commit A - merge B, D, and E
commit B F
    Commit B - after F
commit D F
    Commit D - after F
commit E Z
    Commit E - after Z
commit F G
    Commit F - after I
commit G H I
    Commit F - merge H and I
commit H I
    Commit H - after I
commit I Z
    Commit I - after Z
commit Z
    Commit Z
