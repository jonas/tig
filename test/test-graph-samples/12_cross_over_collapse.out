●─┬─╮ Commit A - merge B, D, and E
● │ │ Commit B - after F
│ ● │ Commit D - after F
│ │ ● Commit E - after Z
●─╯ │ Commit F - after I
●─╭─╯─╮ Commit F - merge H and I
● │ ╭─╯ Commit H - after I
●─│─╯ Commit I - after Z
◎─╯ Commit Z
